### PR TITLE
build localstack-client recipe

### PR DESCRIPTION
**Pull request by mwilson8**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by mwilson8** [comment]
@oblute @benhuff do you agree to be maintainers on this package?

File: recipes/localstack-client/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    - boto3
+    - python
+
+test:
+  imports:
+    - localstack_client
+
+about:
+  home: "https://github.com/localstack/localstack-python-client"
+  license: "Apache Software"
```
Comment (marcelotrevisani) [inline recipes/localstack-client/meta.yaml]: It should be 
```suggestion
  license: "Apache-2.0"
```

File: recipes/localstack-client/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{% set name = "localstack-client" %}
+{% set version = "0.11" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "7cf28213ceeb6fb724153ceeacc08bc9d7b5295c9d200b00a7c131bf067b52f4"
```
Comment (marcelotrevisani) [inline recipes/localstack-client/meta.yaml]: Could you please remove the quotes on ``sha256`` ?
```suggestion
  sha256: 7cf28213ceeb6fb724153ceeacc08bc9d7b5295c9d200b00a7c131bf067b52f4
```

File: recipes/localstack-client/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/localstack-client/meta.yaml]: Could you please change it to:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/localstack-client/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 7cf28213ceeb6fb724153ceeacc08bc9d7b5295c9d200b00a7c131bf067b52f4
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (scopatz) [inline recipes/localstack-client/meta.yaml]: Can this be `noarch: python`

File: recipes/localstack-client/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+requirements:
+  host:
+    - boto3
```
Comment (scopatz) [inline recipes/localstack-client/meta.yaml]: Does this need to be in host at all

File: recipes/localstack-client/meta.yaml (hunk 4 above)
Comment (mwilson8) [reply recipes/localstack-client/meta.yaml]: yes, thank you

File: recipes/localstack-client/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+  license_file: "LICENSE"
+  summary: "A lightweight Python client for LocalStack."
+  doc_url: "https://pypi.org/project/localstack-client/"
+  dev_url: "https://github.com/localstack/localstack-python-client"
+
+extra:
+  recipe-maintainers:
+    - mwilson8
+    - oblute
+    - benhuff
```
Comment (scopatz) [inline recipes/localstack-client/meta.yaml]: @benhuff @oblute - are you OK with being maintainers on this?

**Comment by mwilson8** [comment]
@scopatz @marcelotrevisani this is ready for final review and merge

**Pull request merged**